### DISCUSSION TITLE
[Security] Implementing 2FA to access Airbyte Account

### DISCUSSION THREAD

**Discussion by KevinConseilxyz** [airbytehq/airbyte#33933] [Connector Ideas and Features]
Hi Everyone,

It would be nice to introduce a 2FA to access our Airbyte Account (check how Snowflake do it with Duo Mobile). 

This is becoming quite high prio in European countries. 

Best,
Kevin

**Comment by marcosmarxm** [comment]
> This is becoming quite high prio in European countries.

Can you share what law or more information (article, etc) about the topic?

**Comment by KevinConseilxyz** [comment]
@marcosmarxm as of now i do not think it is a law, however our IT security consultants and our legal consultant are asking us to always:
- Enable 2FA when available
- Prioritize platforms supporting 2FA

I know they follow what the German government is recommending (pretty basic) which is available here: https://www.bsi.bund.de/DE/Themen/Verbraucherinnen-und-Verbraucher/Informationen-und-Empfehlungen/Cyber-Sicherheitsempfehlungen/Accountschutz/Zwei-Faktor-Authentisierung/zwei-faktor-authentisierung_node.html

[a comment by an automated account is not shown]
